apps/storybook/docs/b_bijdragen/3_git-branching-strategie.stories.mdx

```diff
@@ -121,11 +121,13 @@ van de branch vermeld, approvers kunnen dan snel verifieren dat de build in orde
 
 ## Team Branching Strategie
 
-Artefacts worden enkel gebouwd voor pre-release en release branches. Teams die ontwikkeling doen aan de component
-bibliotheek doen dat meestal in de context van aanpassingen voor een concrete eindtoepassing. Vaak is dat werk
-experimenteler van aard, om dat in goede banen te leiden is er per team een branch voorzien waaruit ook
-artefacts gegenereerd worden. Op die manier kunnen ze integreren met eigen beta-artefacts, pas als hun werk
-definitief is moeten ze squashen, rebasen en een PR indienen om hun code te integreren.
+Artefacts worden enkel gebouwd voor de develop en release branches.
+
+Teams die ontwikkeling doen aan de component bibliotheek doen dat meestal in de context van aanpassingen voor een
+concrete eindtoepassing. Vaak is dat werk experimenteler van aard, je kan daarvoor ook beta-artifacts genereren zodat
+je kan integreren, je dient daarvoor de code te integreren met een branch die begint met `develop`, bvb. `develop-xyz`.
+Op die manier kan de code afgenomen en geverifieerd worden, als het werk definitief is moet het wel nog steeds
+ge-squashed, rebased en via een PR geintegreerd worden!
```

Comment:
`gesquasht`, `gerebased`

Reply:
Kan je dit eventueel ook nog toevoegen (n eventueel sub titel voor maken). Is iets dat ik soms moet uitleggen en is handig als ik kan linken naar een specifieke sectie in onze storybook.

`Snapshot releases gemaakt voor alle branches die beginnen met `develop-`. De relevante artifacts kan je terugvinden op [artifactory (snapshot-npm)](https://repo.omgeving.vlaanderen.be/ui/repos/tree/General/snapshot-npm).
Je dient van alle artifacts dezelfde versie af te nemen.`

Reply:
gesquasht - Werkwoord 1. voltooid deelwoord van squashen (de racketsport)
dat is in elk geval niet wat ik hier bedoel: een vernederlandsing van het engelse git commando
en of gerebased correcter is dan rebased weet ik ook niet